mod_reforged/hooks/events/events/dlc4/kings_guard_2_event.nut

```diff
@@ -0,0 +1,15 @@
+::Reforged.HooksMod.hook("scripts/events/events/dlc4/kings_guard_2_event", function(q) {
+	q.create = @(__original) function()
+	{
+		__original();
+		local start = this.m.Screens[0].start;
+		this.m.Screens[0].start = function( _event )
+		{
+			start(_event);
+			local kingsguard_perkTree = ::new("scripts/skills/backgrounds/kings_guard_background").m.PerkTree;
+			kingsguard_perkTree.setActor(_event.m.Dude);
+			kingsguard_perkTree.build();
+			_event.m.Dude.getPerkTree().merge(kingsguard_perkTree);
```

Comment:
From a design pov the kingsguard should be granted one full perk reset at this point in the event. Just like if he had Promised Potential.
Otherwise the correct play is to not spend a single perk for 30+ days until this event proccs, so you don't miss out on stronger perks

Reply:
Good point. I will add that